tools/fpgabist/bist_common.py

```diff
@@ -32,9 +33,21 @@
 
 # TODO: Use AFU IDs vs. names of AFUs
 BIST_MODES = ['bist_afu', 'dma_afu', 'nlb_mode_3']
+REQ_CMDS = ['lspci', 'fpgainfo', 'fpgaconf', 'fpgadiag', 'fpga_dma_test',
+            'bist_app']
 
 
 # Return a list of all available bus numbers
+def check_commands():
+    terminate = False
+    for cmd in REQ_CMDS:
+        executable = find_executable(cmd, path=os.environ['PATH'])
```

Comment:
By default, if `path` is not provided, `find_executable` function uses `os.environ['PATH']`